gdpr/impl.go

```diff
@@ -2,11 +2,13 @@ package gdpr
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/prebid/go-gdpr/api"
 	tcf1constants "github.com/prebid/go-gdpr/consentconstants"
 	consentconstants "github.com/prebid/go-gdpr/consentconstants/tcf2"
 	"github.com/prebid/go-gdpr/vendorconsent"
+	tcf2 "github.com/prebid/go-gdpr/vendorconsent/tcf2"
```

Comment:
Does this need an alias defined? Wouldn't it just be tcf2 without it?

Reply:
No, it would be `vendorconsent` if undefined, and conflict with the line above.

Reply:
Oh, ok.